README.md

```diff
@@ -60,9 +62,9 @@ Alpha tasks:
 
 ### Beta
 
-The goal of the Beta release is to communicate that the SDK is ready for broader consumption, after we believe the APIs are stable enough and we have all the supporting resources in place (website, documentation, examples, etc
+The goal of the Beta release is to make the SDK available for broader consumption, after the APIs are stable and we have all supporting resources in place (website, documentation, examples, and so on)
```

Comment:
Beta doesn’t mean the API is stable. It means we don’t expect major changes.

Reply:
Suggested a change for now, and we can revisit exact phrasing with Hilary once she's back